libreflect/src/stack_setup.c

```diff
@@ -17,6 +18,10 @@
 // none. Requires 18 * size_of(size_t) bytes of memory.
```

Comment:
Does this line need updated now

Reply:
Yes! Comment should be updated to state `20 * size_of(size_t)`. I pushed an update to the branch to adjust the comment.